src/components/SubmissionConfirmation.vue

```diff
@@ -1,32 +1,70 @@
 <template>
-    <div align="center">
-        <v-container style="background-color: #ecf0f1; max-width: 400px">
-            <v-container style="background-color: #c1c1c1; max-width: 400px">
-                <h3><b>Are you sure you want to submit?</b></h3>
+    <div align="center" class="pa-5">
+        <v-dialog align="center" v-model="dialog" width="500">
+            <template v-slot:activator="{ on, attrs }">
+                <v-btn color="primary" dark v-bind="attrs" v-on="on">
+                    Submit
+                </v-btn>
+            </template>
+            <v-container style="background-color: #ecf0f1; max-width: 500px">
+                <v-container
+                    style="background-color: #c1c1c1; max-width: 500px"
+                >
+                    <h3><b>Are you sure you want to submit?</b></h3>
+                </v-container>
+                <p style="padding: 15px 0px">
+                    You will not be able to redo any questions after you submit.
+                </p>
+                <v-btn
+                    style="
+                        margin-right: 10px;
+                        color: white;
+                        background-color: #09b683;
+                    "
+                    x-large
+                    to="/submission"
+                    @click="submitQuiz"
```

Comment:
I think the to="/submission" is going to cause the @click to be ignored, could potentially use router redirect in the submitQuiz method.
E.g this.$router.push('/submission')